hcipy/optics/deformable_mirror.py

```diff
@@ -76,7 +76,7 @@ def make_gaussian_influence_functions(pupil_grid, num_actuators_across_pupil, ac
     ModeBasis
         The influence functions for each of the actuators.
     '''
-    actuator_positions = make_actuator_positions(num_actuators_across_pupil, actuator_spacing)
+    actuator_positions = make_actuator_positions(num_actuators_across_pupil, actuator_spacing, x_tilt, y_tilt, z_tilt)
```

Comment:
No. The `actuator_positions` are the positions in the untiled plane. The Gaussian pokes get calculated on the untiled plane first, then transformed by the `transform_poke()` function to a tiled plane. So the current code is correct.

Unless I'm mistaken in this assessment?